src/App.js

```diff
@@ -36,6 +40,8 @@ class App extends Component {
       });
 
       this.setState({
+        currentUsStateName: 'default',
```

Comment:
Thank you for adding this in to showVisitedParks and ShowBucketList to clear out any state information if it was showing previsouly